docs/getting-started/outreach.md

```diff
@@ -76,6 +78,10 @@ Talks given about Gutenberg, including slides and videos as they are available.
 -   [Anatomy of a block: Gutenberg design patterns](https://wordpress.tv/2018/07/08/tammie-lister-anatomy-of-a-block-gutenberg-design-patterns/), Tammie Lister (July 2018)
 -   [State of the Word 2017 Gutenberg Demo](https://youtu.be/XOY3ZUO6P0k?t=2100), Matt Mullenweg with demo by Matías Ventura Bausero (December 2017)
 -   [Gutenberg is Coming (Don’t Be Afraid)](https://training.ithemes.com/webinar/gutenberg-is-coming-dont-be-afraid/), iThemes Training (September 2017)
+-   [Converting a Shortcode into a Block](https://learn.wordpress.org/tutorial/converting-a-shortcode-into-a-block/), Jonathoan Bossenger
+-   [Using the create-block Tool](https://learn.wordpress.org/tutorial/using-the-create-block-tool/), Jonathoan Bossenger
+-   [Ryan Welcher Codes](https://www.youtube.com/@RyanWelcherCodes), YouTube Channel
```

Comment:
I don't think it makes sense to add people's youtube channels here. I'd recommend removing. There are MANY YouTubers otherwise.

Reply:
Are there many who focus on block development to the extent that Ryan does?

Reply:
@annezazu is the concern the optics around who is selected to be on the list?

Reply:
The concern is around a combination of optics and impact on revenue for YouTubers. I think it's okay to link to individual videos or series. It's the same as linking to someone's entire blog rather than being more specific in my mind.

Reply:
>  It's the same as linking to someone's entire blog rather than being more specific in my mind.

That's a pretty good analogy 🤔. I can totally see your point of view. But I can also see the value of providing ready access to the kind of work that Ryan (and others) are doing in their channels. 

I'd love to get other people's thoughts on this so we can weigh up both sides of the argument and get this call right.